### DISCUSSION TITLE
Searching through multiple tables at once?

### DISCUSSION THREAD

**Discussion by aws022** [Budibase/budibase#12337] [Help]
I have multiple tables in my database, I'd like to either
1.  Search them all simultaneously for an entry
or
2. Aggregate my tables into one large table, so I can search just this table instead

Any pointers?
Thank you

**Comment by melohagan** [comment]
Hey @aws022 

You could potentially use [sync automations](https://docs.budibase.com/docs/synchronous-automations) to do this.

Have an automation that runs a couple of **Query Rows** actions, passing in values you want to filter on into an [App action](https://docs.budibase.com/docs/app-action)

Then you can aggregate and return the combined results in a final **Collect Data** step.
